setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+## change directory to backend
+cd backend
+
+## install the backend dependencies
+echo "installing backend dependencies..."
```

Comment:
Setup script should do the following - 
- Installing dependencies
- Starting docker container
- Migrations and Seeding database